src/components/Episodes/renderUtils.jsx

```diff
@@ -4,25 +4,28 @@ import Episode from "COMPONENTS/Episode/";
 function renderEpisode(data, index){
 
 	const { 
-		number,
+		episodeNo,
 		title,
 		synopsis,
-		image,
-		video,
+		thumbnail,
+		trailer,
 	} = data;
 
-	const id = `episode__${number}`;
+	const id = title
+			.replace(/\W+/g, '-') // remove non-alphanumeric & replace with -s
+			.toLowerCase()
+			;
 
 	return (
 		<Episode
 			id={ id }
 			key={ id }
 			index={ index }
-			number={ number }
+			episodeNo={ episodeNo }
```

Comment:
ah, we might not even need to alias out of the data-object, as we're giving them straight as props, which will alias them from here on out - we can just do `episodeNo={number}` here if we wanted to.

Reply:
Changed back to number={number}. Then destructuring out as episodeNo in EpisodePreview.  Makes more sense in there as there will be a reference to prev and next episodes. Depending on implementation of prev/next episode ids (for next and previous with overlay open), might need to revisit this.